src/fetchtastic/download/firmware.py

```
@@ -2035,6 +2083,24 @@ def download_repo_prerelease_firmware(
+            self._sort_prerelease_dirs(successful_dirs)[-1] if successful_dirs else None
+        )
+        if latest_successful_dir and self.config.get(
+            "CREATE_LATEST_SYMLINKS", DEFAULT_CREATE_LATEST_SYMLINKS
+        ):
+            try:
+                update_latest_pointer(
+                    prerelease_base_dir,
+                    latest_successful_dir,
+                    LATEST_POINTER_NAME,
+                )
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)